### DISCUSSION TITLE
i wanted to add backend to this project

### DISCUSSION THREAD

**Discussion by trahulprabhu38** [ANSHIKA-26/WordWise#234] [Ideas]
If i can make a backend code for this project and call for PR then how will it work? like this is a deployed project so will you deploy backend on heroku ?

**Comment by ANSHIKA-26** [comment]
what tech stacks do you plan to use for backend?

**Comment by trahulprabhu38** [comment]
express and mongoDB

**Reply by ANSHIKA-26** [reply]
sound good , feel free to use schema of your choice !

**Comment by ANSHIKA-26** [comment]
once the backend is made , we can host it on render and then connect it with frontend

**Reply by trahulprabhu38** [reply]
suree ill try once.

**Reply by ANSHIKA-26** [reply]
go ahead , all the best !
